lib/slax_web/websockets/issue.ex

```diff
@@ -11,24 +12,28 @@ defmodule SlaxWeb.Issue do
   def handle_event(%{"bot_id" => bot_id}) when not is_nil(bot_id), do: nil
 
   def handle_event(%{"thread_ts" => ts, "channel" => channel, "text" => text, "type" => "message"}) do
-    issues_scan = scan_text_for_issue(text)
-    prs_scan = scan_text_for_pr(text)
+    unless !is_nil(Channels.get_by_channel_id(channel)) and Channels.disabled?(channel) do
```

Comment:
I'm wondering if it's best to move this catch before the data gets sent to the handlers. Like maybe in the websocket_listener